x-pack/plugins/observability_solution/inventory/server/routes/entities/route.ts

```diff
@@ -4,31 +4,28 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { i18n } from '@kbn/i18n';
-import type { EntityTypeDefinition } from '../../../common/entities';
 import { createInventoryServerRoute } from '../create_inventory_server_route';
+import { createEntitiesESClient } from '../../lib/create_es_client/create_entities_es_client';
+import { getLatestEntities } from './get_latest_entities';
 
-export const listEntityTypesRoute = createInventoryServerRoute({
-  endpoint: 'GET /internal/inventory/entity_types',
+export const listLatestEntitiesRoute = createInventoryServerRoute({
+  endpoint: 'GET /internal/inventory/entities',
```

Comment:
Should we try to get rid off the `/internal/` path on new stuff?

Reply:
According to the [API guideline](https://github.com/elastic/kibana/pull/192570), if this is intended to be an internal API, we should keep it.

Reply:
Why?

Reply:
Oh, sorry! I got confused by [this documentation about doing some changes in internal API access](https://github.com/elastic/kibana/issues/186781) and may have misunderstood something.